src/server/pkg/obj/amazon_client.go

```diff
@@ -3,34 +3,45 @@ package obj
 import (
 	"fmt"
 	"io"
+	"net/http"
+	"strings"
+	"time"
 
+	log "github.com/Sirupsen/logrus"
```

Comment:
Nope, was just using what other files in this pkg were using. Updated this and obj.go to use lion as well